packages/fiori-generator-shared/src/telemetry/telemetryHelper.ts

```diff
@@ -0,0 +1,103 @@
+import {
+    PerformanceMeasurementAPI as Performance,
+    initTelemetrySettings,
+    type TelemetryProperties,
+    type ToolsSuiteTelemetryInitSettings
+} from '@sap-ux/telemetry';
+import type { TelemetryData } from './types';
+import { getHostEnvironment } from '../environment';
+import osName from 'os-name';
+import { t } from '../i18n';
+
+/**
+ * Helper class for intialising and preparing event data for telemetry.
+ */
+export abstract class TelemetryHelper {
```

Comment:
Nothing specific to Fiori generators in here, would this be better situated in `@sap-ux/telemetry` for re-use?

Reply:
We chatted about this on a previous call, I was of the understanding that because this is just a helper class which is used/shared across the sub generators, it was the appropriate place for it.